backend/hub/hub.go

```diff
@@ -52,26 +56,27 @@ type hub struct {
 	gameFactory gamefactory.GameFactory
 	gameRouter  game.GameRouter
 
-	reg registry.Registry
-}
-
-// hubClient should only store basic information about the client
-// and the send channel
-type hubClient struct {
-	send     (chan []byte)
-	nickname string
+	reg         registry.Registry
+	tokenSecret []byte
 }
 
 func GetEmptyHub() Hub {
 	return &hub{}
 }
 
-func GetHub(gameFactory gamefactory.GameFactory) Hub {
+func GetHub(gameFactory gamefactory.GameFactory) (Hub, error) {
+	secret := make([]byte, 60)
```

Comment:
magic number here... but basically the secret rand.Read (from crypt/rand - which is cryptographically safe) is 60 in length. So it's a pretty strong password?